site/docs/mark/mark.md

```diff
@@ -125,11 +125,16 @@ The rest of this section describe groups of properties supported by the `mark` c
 
 {% include table.html props="opacity,fillOpacity,strokeOpacity" source="MarkConfig" %}
 
-
 ### Stroke Style
 
 {% include table.html props="strokeWidth,strokeDash,strokeDashOffset" source="MarkConfig" %}
 
+### Link Properties
+
+Marks can act as hyperlinks when the `href` property is defined.
```

Comment:
- `href` property -> `href` channel

- Maybe also add:

A `cursor` channel can also be provided to serve as affordance for the links.

Reply:
Done